src/app/components/Core/TimeElapsed.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+import { formatDistanceToNow } from 'date-fns';
+
+import { useInterval } from '@app/hooks';
+
+const TimeElapsed = ({ value }: { value: number }) => {
+  const formatTimeDifference = (val: number) =>
```

Comment:
Maybe move this to a separate test file.

Reply:
What do you mean?

Reply:
Sorry, I meant separate file, and add some tests to it. That makes it easier to swap `date-fns` out with something else, if we ever need to do so.